src/style.css

```diff
@@ -1,3 +1,76 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, system-ui;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, you could specify a generic font family as a fallback: `font-family: Roboto, Arial, sans-serif;`.